module/Makefile.in

```
@@ -57,6 +57,7 @@ modules-Linux:
 		$(if @KERNEL_LD@,LD=@KERNEL_LD@) $(if @KERNEL_LLVM@,LLVM=@KERNEL_LLVM@) \
 		$(if @KERNEL_CROSS_COMPILE@,CROSS_COMPILE=@KERNEL_CROSS_COMPILE@) \
 		$(if @KERNEL_ARCH@,ARCH=@KERNEL_ARCH@) \
+		$(if @OBJTOOL_DISABLE_WERROR@,objtool=@top_srcdir@/scripts/objtool-wrapper) \
```

Since `objtool-wrapper` is generated from `objtool-wrapper.in` it'll be in the build dir.  Sorry, I missed that the first time around. 

```suggestion
		$(if @OBJTOOL_DISABLE_WERROR@,objtool=@top_builddir@/scripts/objtool-wrapper) \
```

```
[behlendorf@pip zfs]$ mkdir build
[behlendorf@pip zfs]$ cd build/
[behlendorf@pip build]$ ../configure
...
[behlendorf@pip build]$ ls scripts/
objtool-wrapper
```

---

A bit unexpected, at least for me, but makes sense since it's kinda build from the the .in.